### PR TITLE
move instascale source to project

### PR DESCRIPTION
**Pull request by asm582**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,79 @@
+module github.ibm.com/asmalvan/instascale-1
```
Comment (Maxusmusti) [inline go.mod:1]: Needs to be adjusted for new home

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+go 1.17
+
+require (
+	github.com/IBM/multi-cluster-app-dispatcher v0.0.0-20220426132040-4cb2f1b93df4
```
Comment (Maxusmusti) [inline go.mod:6]: Same here

**Review by Maxusmusti — CHANGES_REQUESTED** [review]
Looks good, just need to make sure to adjust any old IBM references

File: go.mod (hunk 1 above)
Comment (asm582) [reply go.mod]: Done, Thanks

File: go.mod (hunk 2 above)
Comment (asm582) [reply go.mod]: Done, Thanks

**Review by Maxusmusti — APPROVED** [review]

**Pull request merged**